src/IRoute.php

```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+
+namespace FastRoute;
+
+interface IRoute
```

Comment:
I'd avoid using prefixes too. This should be simply `Route` and the implementation can live in another namespace (and also use `Route` as name).

The main point is that the interface should be the connection point everywhere.

Reply:
I would prefer a suffix, the full word "Interface" just by looking the files its always a PITA to identify interfaces if there is no prefix or suffix. So I would vote vor changing it everywhere to add a prefix or suffix.

Reply:
That goes against what we have in the project. Nothing uses prefixes nor suffixes because we don't see value in differentiating these components in this way.